tests/data/program_analysis/select_case/select02.f

```diff
@@ -0,0 +1,34 @@
+C FORTRAN test file to test CASE constructs
+C This file tests an integer with various types of comparisons
+
+      PROGRAM MAIN
+
+      IMPLICIT NONE
+
+      INTEGER :: I = 5
+      INTEGER :: X = 40
+      INTEGER :: Y
+      INTEGER :: Z = 2
+
+      SELECT CASE(I)
```

Comment:
Would it be possible to add a loop around the `SELECT` statement so that all of the case labels are exercised?

Reply:
Modified the test file.